presto-main/src/test/java/com/facebook/presto/type/TestArrayOperators.java

```diff
@@ -160,6 +163,32 @@ public void testArrayElements()
         assertInvalidFunction("CAST(ARRAY ['puppies', 'kittens'] AS ARRAY<BIGINT>)", INVALID_CAST_ARGUMENT);
     }
 
+    private static String generateLongSring(int size, char c)
+    {
+        char[] cs = new char[size];
+        Arrays.fill(cs, c);
+
+        return new String(cs);
+    }
+
+    private static String generateThreeElementArrayString(int size)
```

Comment:
the name of the function wasn't why i made it, btw
I wanted a separate variable, and i also had much more complicated code to generate the string before. I think now with String.repeat() it's simple enough that it doesn't warrant a function (it had a loop and StringBuilders and more).

The name is kind of pointless--it was just a helper function to help keep the core of the test from being cluttered

I'll update and inline.